tools/java/org/unicode/cldr/util/data/language_script_raw.txt

```diff
@@ -220,8 +219,8 @@ ewo	Ewondo	primary	Latn	Latin
 ext	Extremaduran	primary	Latn	Latin
 fa	Persian	primary	Arab	Arabic
 fan	Fang	primary	Latn	Latin
+ff	Fulah	primary	Adlm	Adlam
```

Comment:
Again, this is not consistent with the locale data, in which "ff" defaults to Latn,